engine/include/cubos/engine/plugins/renderer.hpp

```diff
@@ -0,0 +1,31 @@
+#ifndef CUBOS_ENGINE_PLUGINS_RENDERER_HPP
+#define CUBOS_ENGINE_PLUGINS_RENDERER_HPP
+
+#include <cubos/engine/ecs/transform_system.hpp>
+
+#include <cubos/engine/cubos.hpp>
+
+#include <glm/glm.hpp>
+#include <cubos/core/gl/render_device.hpp>
+#include <cubos/core/ecs/entity_manager.hpp>
+
+namespace cubos::engine::plugins
+{
+    /// Plugin to create and use a renderer.
+    ///
+    /// Startup Stages:
```

Comment:
Rename to tags (and fix other cases in `transform.hpp` too)
```suggestion
    /// Startup Tags:
```